inc/plugins/hello.php

```diff
@@ -8,17 +8,58 @@
  *
  */
 
-// Disallow direct access to this file for security reasons
-if(!defined("IN_MYBB"))
+// Make sure we can't access this file directly from the browser.
+if(!defined('IN_MYBB'))
 {
-	die("Direct initialization of this file is not allowed.<br /><br />Please make sure IN_MYBB is defined.");
+	die('This file cannot be accessed directly.');
 }
+	
+// cache templates - this is important when it comes to performance
+// THIS_SCRIPT is defined by some of the MyBB scripts, including index.php
+if(defined('THIS_SCRIPT'))
+{
+    global $templatelist;
+
+    if(isset($templatelist))
+    {
+        $templatelist .= ',';
```

Comment:
Wouldn't this break something by adding a comma when it might not be used?
Better to have a comma on the two lines below instead. Also there shouldn't be spaces for consistency.
```php
	if(THIS_SCRIPT== 'index.php')
	{
		$templatelist .= ',hello_index,hello_message';
	}
	elseif(THIS_SCRIPT== 'showthread.php')
	{
		$templatelist .= ',hello_post,hello_message';
	}```